scania-digital-design-system/tegel
# feature: adding motion to button component

## **Describe pull-request**  
In this PR we are adding animation to the button component background color

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** Add ticket number after `CDEP-3709`: [CDEP-3709](https://tegel.atlassian.net/browse/CDEP-3709)    


## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to the AWS link. 
2. Navigate to the Button component
3. Enable animation fade
4. Make sure the button now has an small animation for the background color

## **Checklist before submission**

## **Suggested test steps**

## **Screenshots**  
Below is the animation which we should apply on the button:
![Screenshot 2024-12-18 at 16 00 39](https://github.com/user-attachments/assets/c428dff9-93e2-4b5f-867e-a91cfd4f131f)

[CDEP-3709]: https://tegel.atlassian.net/browse/CDEP-3709?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
